### PR TITLE
tiem: update tiem docs for v1.0.2 open source

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
tiem: update tiem docs for v1.0.2 open source

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by haiboumich — APPROVED** [review]
LGTM

File: download-ecosystem-tools.md [hunk 1]
```diff
@@ -5,85 +5,57 @@ aliases: ['/docs-cn/dev/download-ecosystem-tools/','/docs-cn/dev/reference/tools
...
-使用下表中的链接下载 [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md):
-
-| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
-|:---|:---|:---|:---|
-| [tidb-enterprise-tools-nightly-linux-amd64.tar.gz](https://download.pingcap.org/tidb-enterprise-tools-nightly-linux-amd64.tar.gz) | Linux | amd64 | [tidb-enterprise-tools-nightly-linux-amd64.sha256](https://download.pingcap.org/tidb-enterprise-tools-nightly-linux-amd64.sha256) |
-
-## TiCDC
-
-[TiCDC](/ticdc/ticdc-overview.md) 的下载方式见 [TiCDC 安装部署文档](/ticdc/deploy-ticdc.md)。
+> 下载链接中的 `{version}` 为 TiEM 的版本号。例如，`v1.0.2` 版本的下载链接为 `https://download.pingcap.org/tidb-toolkit-v1.0.2-linux-amd64.tar.gz`。
```
Comment (TomShawn) [inline download-ecosystem-tools.md:61]:
```suggestion
> - 下载链接中的 `{version}` 为 TiEM 的版本号。例如，`v1.0.2` 版本的下载链接为 `https://download.pingcap.org/tidb-toolkit-v1.0.2-linux-amd64.tar.gz`。
> - TiEM 是从 v1.0.2 起开放源代码的，因此下载链接中 `{version}` 支持的最低版本为 `v1.0.2`。你不能将 `{version}` 替换为 `v1.0.0` 或 `v1.0.1`。
```

File: download-ecosystem-tools.md [hunk 2]
```diff
@@ -5,85 +5,58 @@ aliases: ['/docs-cn/dev/download-ecosystem-tools/','/docs-cn/dev/reference/tools
...
-> 下载链接中的 `{version}` 为 DM 的版本号。例如，`v6.0.0` 版本的下载链接为 `https://download.pingcap.org/dm-v6.0.0-linux-amd64.tar.gz`。可以通过 [DM Release](https://github.com/pingcap/tiflow/releases) 查看当前已发布版本。
+| 工具  | 离线包名称  |
+|:------|:----------|
+| [TiUP](/tiup/tiup-overview.md)  | `tiup-linux-amd64.tar.gz` <br/>`tiup-{tiup-version}-linux-amd64.tar.gz` <br/>`dm-{tiup-version}-linux-amd64.tar.gz`  |
+| [Dumpling](/dumpling-overview.md)  | `dumpling-{version}-linux-amd64.tar.gz`  |
+| [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)  | `tidb-lightning-ctl` <br/>`tidb-lightning-{version}-linux-amd64.tar.gz`  |
+| [TiDB DM (Data Migration)](/dm/dm-overview.md)  | `dm-worker-{version}-linux-amd64.tar.gz` <br/>`dm-master-{version}-linux-amd64.tar.gz` <br/>`dmctl-{version}-linux-amd64.tar.gz`  |
+| [TiCDC](/ticdc/ticdc-overview.md)  | `cdc-{version}-linux-amd64.tar.gz`  |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)  | `pump-{version}-linux-amd64.tar.gz` <br/>`drainer-{version}-linux-amd64.tar.gz` <br/>`binlogctl` <br/>`reparo`  |
+| [Backup & Restore (BR)](/br/backup-and-restore-overview.md)  | `br-{version}-linux-amd64.tar.gz`  |
```
Comment (TomShawn) [inline download-ecosystem-tools.md:45]:
```suggestion
| [Backup & Restore (BR)](/br/backup-and-restore-tool.md)  | `br-{version}-linux-amd64.tar.gz`  |
```

File: download-ecosystem-tools.md [hunk 3]
```diff
@@ -5,85 +5,58 @@ aliases: ['/docs-cn/dev/download-ecosystem-tools/','/docs-cn/dev/reference/tools
...
-> 下载链接中的 `{version}` 为 DM 的版本号。例如，`v6.0.0` 版本的下载链接为 `https://download.pingcap.org/dm-v6.0.0-linux-amd64.tar.gz`。可以通过 [DM Release](https://github.com/pingcap/tiflow/releases) 查看当前已发布版本。
+| 工具  | 离线包名称  |
+|:------|:----------|
+| [TiUP](/tiup/tiup-overview.md)  | `tiup-linux-amd64.tar.gz` <br/>`tiup-{tiup-version}-linux-amd64.tar.gz` <br/>`dm-{tiup-version}-linux-amd64.tar.gz`  |
+| [Dumpling](/dumpling-overview.md)  | `dumpling-{version}-linux-amd64.tar.gz`  |
+| [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)  | `tidb-lightning-ctl` <br/>`tidb-lightning-{version}-linux-amd64.tar.gz`  |
+| [TiDB DM (Data Migration)](/dm/dm-overview.md)  | `dm-worker-{version}-linux-amd64.tar.gz` <br/>`dm-master-{version}-linux-amd64.tar.gz` <br/>`dmctl-{version}-linux-amd64.tar.gz`  |
+| [TiCDC](/ticdc/ticdc-overview.md)  | `cdc-{version}-linux-amd64.tar.gz`  |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)  | `pump-{version}-linux-amd64.tar.gz` <br/>`drainer-{version}-linux-amd64.tar.gz` <br/>`binlogctl` <br/>`reparo`  |
+| [Backup & Restore (BR)](/br/backup-and-restore-tool.md)  | `br-{version}-linux-amd64.tar.gz`  |
```
Comment (TomShawn) [inline download-ecosystem-tools.md:45]:
```suggestion
| [Backup & Restore (BR)](/br/backup-and-restore-overview.md)  | `br-{version}-linux-amd64.tar.gz`  |
```

File: tiem/tiem-overview.md [hunk 4]
```diff
@@ -110,7 +112,7 @@ TiDB 自身采用多副本的数据存储，TiEM 根据调度策略将 TiDB 的
 
 ### 集群参数管理
 
-- 提供 TiDB 版本对应参数组模板（TiEM 1.0.0 支持 TiDB 5.0.x， 5.1.x， 5.2.x， 5.3.x， 5.4.x 参数，暂不支持 TiDB 6.0 参数）
+- 提供 TiDB 版本对应参数组模板。TiEM 支持的 TiDB 版本，见 [TiEM 支持的 TiDB 版本](/tiem/tiem-release-notes.md#tiem-支持的-tidb-版本)。
```
Comment (lilin90) [inline tiem/tiem-overview.md:115]:
```suggestion
- 提供 TiDB 版本对应的参数组模板。关于支持的 TiDB 版本，参见 [TiEM 支持的 TiDB 版本](/tiem/tiem-release-notes.md#tiem-支持的-tidb-版本)。
```

File: tiem/tiem-faq.md [hunk 5]
```diff
@@ -17,15 +17,9 @@ TiEM 管理的集群可以部署在用户自建的数据中心或公有云。
...
-TiEM v1.0.0 支持以下 TiDB 版本：
-
-- TiDB v5.0.x
-- TiDB v5.1.x
-- TiDB v5.2.x
-- TiDB v5.3.x
-- TiDB v5.4.x
+见 [TiEM 支持的 TiDB 版本](/tiem/tiem-release-notes.md#tiem-支持的-tidb-版本)。
 
 ### TiEM 与 TiUP 的关系是什么？
```
Comment (lilin90) [inline tiem/tiem-faq.md:25]: 第 26 行“完成……功能”的表述可以稍微调整优化下，例如将”完成“改成”提供“或”实现“：

`TiEM 使用 TiDB 及其生态工具、API 实现数据库管理功能，TiUP 是 TiEM 使用的工具之一。TiEM 的架构图如下：`

Comment (TomShawn) [reply tiem/tiem-faq.md:25]: 改为了“提供”。

**Review by zhoubasten — APPROVED** [review]
LGTM

**Pull request merged**